components/Academy/AcademyDayCards/AcademyDayCards.module.scss

```diff
@@ -99,6 +99,11 @@
     margin: 0 auto;
 }
 
+.postItNotePerson {
```

Comment:
Non-essential for this MR, but it might be worth creating a shared 'person' class for these and just having variations for e.g. `microphonePerson`

Reply:
Refactored to a shared class, thanks for the review :)